src/Sulu/Bundle/ContentBundle/Resources/public/js/components/smart-content/categories/main.js

```diff
@@ -138,7 +138,7 @@ define(['services/husky/util', 'services/sulucontent/user-settings-manager'], fu
                             instanceName: this.options.instanceName,
                             url: [
                                 '/admin/api/categories',
-                                '?locale=' + UserSettingsManager.getContentLocale(this.options.webspace),
+                                '?locale=' + this.sandbox.sulu.user.locale,
```

Comment:
the problem is that this function uses the webspace (which is not present in article-bundle). How should we solve that? if webspace exists then content-locale else userlocale?